src/libraries/System.Diagnostics.EventLog/src/System/Diagnostics/EventLog.cs

```diff
@@ -265,7 +265,9 @@ public static void CreateEventSource(EventSourceCreationData sourceData)
                 throw new ArgumentException(SR.Format(SR.ParameterTooLong, nameof(source), 254 - EventLogKey.Length));
 
             Mutex mutex = null;
+#if FEATURE_CER
```

Comment:
This file is compiled for .NET Core only. We do not need this under ifdefs. This should be deleted instead.

(multiple places)

Reply:
Ah, that makes sense. I was leaving the calls in place for `netcoreapp2.0`, but since it's a no-op it makes sense that it can be removed.  Thanks!

Reply:
And on that note, I think I can change my pattern for how I define `FEATURE_CER` such that it's only defined when `netstandard` or `net4` are targeted.